DOCUMENTATION.md

```diff
@@ -512,4 +514,22 @@ You can then update the property you want to update using the `set` command:
 
 ```content-cli set connection --profile <profile> --dataPoolId <dataPoolId> --connectionId <connectionId> --property <property> --value <value>```
 
+### Config commands
+
+
+### List all packages across flavors
+
+With this command you can retrieve a list of all packages across flavors within a team.
```

Comment:
For listing, it is possible to use the optional `--flavors` param to filter by flavors, but the default is to return all packages across flavors.
There are no restrictions regarding import/export, but if we want to put such restrictions, I think they belong on package-manager as the CLI, at its core, merely proxies to it.

Reply:
All configurations can be exported from the API, as for the Team To Team copy feature it is up to their service to keep the information of which flavors are enabled and send those in the mentioned param.

Reply:
One thing that still seems a bit weird is that we're exposing a technical detail such as flavor to end users. I don't know the actual usage of this API, but seeing how this can be used from end users, it might be worth considering to rename **flavors** to something else on the CLI side.

Reply:
This API is intended for Team To Team Copy for now and we don't have a plan on whether it should have usage from other users, as it might happen that in the future it becomes internal. 

Because the main usage will be from TTC I think it's fine to have it in the transports as flavor, as for the documentation we can update to say all configurations but that also leaves it unclear for now, as the users will not know what configuration is or for which services you can get it. Did you have a suggestion in mind?

Reply:
I don't have any specific suggestions in mind, maybe something like **configurations** or **configurationTypes**.

> This API is intended for Team To Team Copy for now and we don't have a plan on whether it should have usage from other users, as it might happen that in the future it becomes internal.

To my knowledge, we don't have anything to mark commands on content CLI as internal only. This means that this will be visible to end users if i'm not mistaken.